integration-tests/pom.xml

```diff
@@ -258,5 +258,52 @@
             </dependency>
         </dependencies>
     </dependencyManagement>
+
+    <profiles>
+        <profile>
+            <id>native-image</id>
+            <activation>
+                <property>
+                    <name>native</name>
+                </property>
+                <file>
+                    <exists>${basedir}/run-native-tests</exists>
```

Comment:
Not sure about the name of this flag file. Maybe `enable-native-tests`? Or even `enable-native-profile`?

Reply:
I'm wondering whether it should be the other way around, i.e. `disable-native-profile`, assuming native tests are pretty much a must have.

Reply:
Excellent idea! This will also result in fewer flag files and it all feels a bit more logical when adding custom `systemPropertyVariables`.

Reply:
Done!